src/features/questions/questions-list/FilteredQuestionGroups.tsx

```diff
@@ -21,6 +22,41 @@ const FadeInQuestionGroup = withFadeIn<QuestionGroupItemProps>(
 export const FilteredQuestionGroups: React.FC<{
   filterQuestionGroups: ReadonlyArray<QuestionGroup>
 }> = ({ filterQuestionGroups }) => {
+  const [focusedItem, setFocusedItem] = useState(1)
```

Comment:
I personally like to use index to reason around things like these :)
```suggestion
  const [focusedItemIndex, setFocusedItemIndex] = useState(0)
```

Reply:
It will not working because of checking for a value. 0 is false and it brings some troubles